### ISSUE TITLE
Definition file for tcomb

### ISSUE THREAD

**Issue by gcanti** [gcanti/babel-plugin-tcomb#39]
The new version (v0.3) of this plugin will require a definition file in order to exploit refinements and runtime type introspection.

I'm working on a first draft but I'm not sure what's the best practice. Where can I put the definition file?

- `tcomb`'s' repo
- this repo
- a new dedicated repo

**Comment by gcanti** [comment]
Until we figure this out, I'll push my drafts here https://github.com/gcanti/pantarei/blob/master/tcomb/3.2.2%2B.js

**Comment by volkanunsal** [comment]
The flow-typed repo seems to be emerging as a collection of declaration files.

https://github.com/flowtype/flow-typed

**Comment by ctrlplusb** [comment]
There is also this one https://github.com/marudor/flowInterfaces

**Comment by ctrlplusb** [comment]
Having used flow for a few weeks now I defo think we should consider putting the definitions in the `flow-typed` repo as suggested by @volkanunsal.  It looks like it is maintained by the official Facebook Flow team and will become ___the___ store for definitions.

Will be great for exposure too perhaps?

**Comment by gcanti** [comment]
Yeah, definitely. The current version still needs some adjustments, then we can send a PR.

**Comment by ArmorDarks** [comment]
Definitely should go into the https://github.com/flowtype/flow-typed, or be built-in directly into the `tcomb` itself.

Btw, are https://github.com/gcanti/pantarei/tree/master/tcomb/3.x.x-0.33.x still actual for the latest `tcomb`?

**Comment by ArmorDarks** [comment]
Seems to not work with Flow 0.83.

![image](https://media.giphy.com/media/tpwwhv1BLd31e/200.gif)


Does anyone happen to have actual types for Flow .80+?
